maven/jdk19/cst-windows.yml

```diff
@@ -0,0 +1,11 @@
+schemaVersion: 2.0.0
+
+commandTests:
+  - name: "Check that `java` is present in the PATH and match jdk19"
+    command: "cmd.exe"
+    args: ["/C", "java", "--version"]
+    expectedOutput: ["Temurin-19"]
+  - name: "Check that `java 11` is present in C:/openjdk-11"
+    command: "cmd.exe"
+    args: ["/C", "C:/openjdk-11/bin/java", "--version"]
```

Comment:
Same, should test jdk19 and not jdk11?

Reply:
no, jdk11 used to spawn the agent and match the jdk used on the controller.
and the jdk19 installed in the /tools folder and the path, for default used from the pipeline (with java)